analytics/src/main/AndroidManifest.xml

```diff
@@ -1,3 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.hms.lib.commonmobileservices.analytics">
+
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.WAKE_LOCK" />
+
+    <meta-data
```

Comment:
you do not have to add this line in AndroidManifest.xml file. Please remove this code line.